### PR TITLE
Add New Fields to HubSpot Upload

### PR DESCRIPTION
**Pull request by CalebPena**
What (if any) features are you implementing?
- Send county field to HubSpot.
- Send household size field to HubSpot.
- Send age fields to HubSpot for every household member.

What (if anything) did you refactor?
- Capture HubSpot errors in Sentry.

### CODE REVIEW

**Review by myang1010 — APPROVED** [review]
Code looks good. 
One question, are we gradually moving away from single quotes?

**Review by ivonne-hernandez — APPROVED** [review]

**Comment by CalebPena** [comment]
> Code looks good. One question, are we gradually moving away from single quotes?

Yes, I have a pr to add the black formatter, I am just waiting for a good time to avoid merge conflict. Will merge that pr after the admin redesign pr is merged.

**Pull request merged**
